### DISCUSSION TITLE
作者可以看看这个项目怎么部署吗？现在这个图床有时候不能上传了，下面链接给了一个新的方案

### DISCUSSION THREAD

**Discussion by Inc-out** [x-dr/telegraph-Image#84] [Ideas]
部署的图片挂了，不知道怎么部署

https://github.com/roimdev/roim-picx

**Comment by x-dr** [comment]
挂了一个接口而已  还能用的

**Comment by Inc-out** [comment]
已测试，D1也没错，变量也没错，TG_Channel

无法上传，腾讯可以上传

x-dr ***@***.***> 于 2024年9月7日周六 23:21写道：
